src/intTest/resources/patient/parameters.json

```diff
@@ -0,0 +1,104 @@
+{
+  "parameter": [
+    {
+      "name": "patient",
+      "resource": {
+        "address": [
+          {
+            "line": [
+              "current house name",
+              "current number or road name",
+              "current locality",
+              "current town",
+              "current county"
+            ],
+            "postalCode": "SO15 2GB",
+            "text": "house name, number or road name, locality, town, country",
+            "use": "home"
+          },
+          {
+            "line": [
+              "current house name",
+              "current number or road name",
+              "current locality",
+              "current town",
+              "current county"
+            ],
+            "postalCode": "SO15 2GB",
+            "text": "house old, number or road name, locality, town, country",
+            "use": "old"
+          }
+        ],
+        "birthDate": "1990-01-01T09:33:00",
+        "extension": [
+          {
+            "url": "http://hl7.org/fhir/StructureDefinition/patient-birthPlace",
+            "valueString": "TownOfBirth"
+          },
+          {
+            "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHAIS-DrugsDispensedMarker",
+            "valueBoolean": true
+          },
+          {
+            "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHAIS-ResidentialInstituteCode",
+            "valueString": "ResidentialInstituteCode"
+          },
+          {
+            "url": "https://fhir.nhs.uk/R4/StructureDefinition/Extension-UKCore-NHAIS-GPNotes",
```

Comment:
I'd include the GP notes as a parameter. The notes are the reason for performing the operation and not part of the patient record